cmd/helpers.go

```diff
@@ -7,16 +7,22 @@ import (
 	"bufio"
 	"fmt"
 	"io"
+	"os"
+	"strconv"
 	"strings"
 
 	"github.com/juju/cmd/v3"
 	"github.com/juju/errors"
+
+	"github.com/juju/juju/juju/osenv"
 )
 
 // This file contains helper functions for generic operations commonly needed
 // when implementing a command.
 
-const msg = "\nContinue [y/N]? "
+const yesNoMsg = "\nContinue [y/N]? "
+
+var nameVerificationMsg = "\nTo continue, enter the name of the %s to be destroyed:"[1:]
```

Comment:
You can drop the `\n` at the beginning and `[1:]` at the end. The reason we have the `[1:]` is to cut off the first newline lol

Reply:
lol, copy-paste power :)

Reply:
fixed